File: src/C#/Parcial2/Geometria/Poligono.cs
using System;

public abstract class Poligono
{
    protected int numLados;

    public void setNumLados(int numLados)
    {
        this.numLados = numLados;
    }

    public int getNumLados()
    {
        return numLados;
    }

    public abstract double calcularArea();
    public abstract double calcularPerimetro();
}

File: src/Java/Parcial2/Geometria/Poligono.java
package Java.Parcial2.Geometria;

public abstract class Poligono {
    protected int numLados;

    public void setNumLados(int n) {
        this.numLados = n;
    }

    public int getNumLados() {
        return this.numLados;
    }

    public abstract double calcularArea();

    public abstract double calcularPerimetro();
}
